packages/components/src/range-control/index.js

```diff
@@ -30,13 +30,15 @@ function RangeControl( {
 	min,
 	max,
 	setState,
+	initialValue,
 	...props
 } ) {
 	const id = `inspector-range-control-${ instanceId }`;
 	const currentInputValue = currentInput === null ? value : currentInput;
+	const initialInputValue = ! initialValue ? -1 : initialValue;
```

Comment:
Can you explain why did you pick `-1` rather than `undefined` like it happens in `onChangeValue` function?

Reply:
Should it also ensure that `initialValue` is between `min` and `max`?

Reply:
Originally, I thought that we should make it -1 because undefined would break something. But I did some more looking into it and realized that this would just leave the box blank.

So I made these two changes and made a new commit.